tasks/keepalived_selinux.yml

```diff
@@ -21,14 +21,11 @@
 
 - name: Ensure SELinux packages are installed
   yum:
-    name:
-      - libselinux
-      - libselinux-devel
-      - checkpolicy
-      - policycoreutils-python
+    name: "{{ keepalived_selinux_packages }}"
     state: present
   when:
     - '"keepalived_ping" not in selinux_modules.stdout'
+    - keepalived_selinux_packages is defined
```

Comment:
You can remove that condition. `keepalived_selinux_packages` is defined everywhere.